services/motion/explore/explore.go

```diff
@@ -0,0 +1,620 @@
+// Package explore implements a motion service for exploration. This motion service model is a temporary model
```

Comment:
[note] Let me know if this is enough of a call out or if you'd like a larger/more obvious one and/or JIRA ticket for it

Reply:
@tessavitabile is the package comment above descriptive enough then?